src/emcee/tests/unit/test_blobs.py

```diff
@@ -20,32 +22,73 @@ def __call__(self, params):
 @pytest.mark.parametrize(
     "blob_spec",
     [
-        (True, 5, lambda x: np.random.randn(5)),
-        (True, 0, lambda x: np.random.randn()),
-        (False, 2, lambda x: (1.0, np.random.randn(3))),
-        (False, 0, lambda x: "face"),
-        (False, 2, lambda x: (np.random.randn(5), "face")),
+        (True, False, (5, 3), lambda x: np.random.randn(5, 3)),
+        (True, False, 5, lambda x: np.random.randn(5)),
+        (True, False, 0, lambda x: np.random.randn()),
```

Comment:
Any dimensions of length 1 get squeezed out, so `randn(1)` would produce the same shape.